compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/ComposeSceneTest.kt

```diff
@@ -274,17 +280,20 @@ class ComposeSceneTest {
     @Test(timeout = 5000)
     fun `rendering of clickable`() = renderingTest(width = 40, height = 40) {
         setContent {
-            Box(Modifier.size(20.dp).background(Color.Blue).clickable {})
+            val interactionSource = remember { MutableInteractionSource() }
+            Box(
+                Modifier
+                    .size(20.dp)
+                    .background(Color.Blue)
+                    .clickable(indication = PressTestIndication, interactionSource = interactionSource) {}
```

Comment:
It used to work well without PressTestIndication, right?  Or was it just flaky?

Reply:
> It used to work well without PressTestIndication

No, it fails because hover interaction have a complicated flow, and can not behave as expected when we snapshot it frame by frame. Reaction can be delayed to the next render frame, or to the next coroutine frame.

Right now it works well in the manual tests. I want to look closer to this in the future.